.github/workflows/test_docs.yml

```diff
@@ -10,20 +10,18 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      matrix:
-        python-version: ["3.10"]
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python ${{ matrix.python-version }}
+    - name: Set up Python
       uses: actions/setup-python@v3
       with:
-        python-version: ${{ matrix.python-version }}
+        python-version: "3.10"
```

Comment:
Why do you revert from matrix - I think having the python version on top is nice - but no deal braker

Reply:
> 

Fixes also #5

Reply:
It doesn't feel right to have the matrix-boilerplate for a `1x1` matrix.